zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/RestoreFromBackupTool.java

```diff
@@ -92,6 +116,162 @@ public RestoreFromBackupTool() {
     this.restoreTempDir = restoreTempDir;
   }
 
+  /**
+   * Parse and validate arguments to the tool
+   * @param cl the command line object with user inputs
+   * @return true if the arguments parse correctly; false in all other cases.
+   * @throws IOException if the backup provider cannot be instantiated correctly.
+   */
+  public void parseArgs(CommandLine cl) {
```

Comment:
This `parseArgs` function is long; is it possible to break them up into private functions to improve readability?